indra/newview/llagent.cpp

```diff
@@ -439,8 +439,6 @@ LLAgent::LLAgent() :
     mIsDoNotDisturb(false),
 
     mControlFlags(0x00000000),
-    mbFlagsDirty(false),
-    mbFlagsNeedReset(false),
```

Comment:
This "FlagsDirty" and "FlagsNeedReset" stuff is not helpful, so it is being removed.